icu4c/source/tools/icu-file-utf8-check.py

```diff
@@ -22,6 +22,8 @@
 #  The tool operates recursively on the directory from which it is run.
 #  Only files from the ICU github repository are checked.
 #  No changes are made to the repository; only the working copy will be altered.
+#  The script checks all source files and keeps a non-zero exit code if any of
```

Comment:
nit: For the phrase, "and keeps a non-zero exit code", would "and returns a non-zero exit code" maybe be a slightly more common way to say it? I'm not sure...
(I see the script does have a rc variable, so it does "keep" track of the return code, but a caller of the program might not care I guess.)

Reply:
I like your wording - done.